### DISCUSSION TITLE
File icon creation

### DISCUSSION THREAD

**Discussion by zrksyd** [ScanMountGoat/ssbh_editor#136] [General]
I was making a new file icon but I could not figure out the base color to use to match the existing icons.

**Comment by zrksyd** [comment]
Nvm I figured it out, it's #ffffff but I think that should be mentioned in CONTRIBUTING.md

**Reply by ScanMountGoat** [reply]
The current implementation is using solid white (#ffffffff) icons and tinting them to match the text color. This only supports icons with a single color. Another option I may explore is just having different sets of icons for each color theme. I'll try and finalize the guidelines for SVG icon coloring closer to the next release.
